railties/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt

```diff
@@ -24,7 +24,7 @@
   // "customizations": {},
 
   // Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
```

Comment:
Should we update this line too?

Reply:
Yes you are right. But today I am thinking I will try to solve this issue in the image itself, so I may close this later today. Otherwise, I will update the comment.